src/pages/_app.js

```diff
@@ -0,0 +1,13 @@
+import '@/styles/globals.scss';
+import Head from 'next/head';
+
+export default function App({ Component, pageProps }) {
+  return (
+    <>
+      <Head>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
this could not be in the document too?

Reply:
the presence of the viewport meta tag in the `_document.js` file causes an alert: "Viewport meta tags should not be used in the `<Head>` of `_document.js`". As recommended by the [documentation](https://nextjs.org/docs/messages/no-document-viewport-meta): "Set your viewport meta tag in `pages/_app.js` instead".

Reply:
not getting this error here, but lgtm

![image](https://user-images.githubusercontent.com/75763403/227733844-8c5885f4-dd6d-4f66-9b42-2089a9613fab.png)